src/textual/app.py

```diff
@@ -1966,6 +1968,14 @@ async def _shutdown(self) -> None:
 
         self._print_error_renderables()
 
+        if constants.SHOW_RETURN and self._exit and self._return_value is not None:
```

Comment:
Even if the return value is `None`, shouldn't it be printed?

E.g., I'm debugging an app and the bug makes it exit at the wrong time (but I don't know yet).
I'm expecting to see a return value but I don't see the final panel.
I'll think the issue lies in the `textual run -r` option.
However, if the `None` is shown, I'll be able to tell that my app returned the wrong thing.